pim-tool-back/src/main/java/vn/elca/training/model/entity/Group.java

```diff
@@ -0,0 +1,66 @@
+package vn.elca.training.model.entity;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+@Table(name = "group_table")
+public class Group {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "leader_id")
+    private User leader;
+
+    @OneToMany(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    @JoinTable(name = "GROUP_PROJECT_TABLE",
+            joinColumns = {@JoinColumn(name = "group_id", referencedColumnName = "id")},
+            inverseJoinColumns = {@JoinColumn(name = "project_id", referencedColumnName = "id")})
+    private Set<Project> projectList;
```

Comment:
[S] Always override hashcode() and equals() when using Set to ensure correctness